dags/springer/parser.py

```diff
@@ -115,6 +114,15 @@ def __init__(self) -> None:
         ]
         super().__init__(extractors)
 
+    def get_dois(self, article: ET.Element):
+        source = "./Journal/Volume/Issue/Article/ArticleInfo/ArticleDOI"
+        doi_element = article.find(source)
+        if doi_element is not None:
+            dois = doi_element.text
+            self.logger.msg("Parsing dois for article", dois=dois)
+            self.dois = dois
+            return [dois]
+
```

Comment:
this function will return only one doi, no?

Reply:
@drjova  should we open a new PR and change in all places that it infliences? It is not the first time this place raises questions.

Reply:
No, it's ok for the moment thanks